google-cloud-bigquery/src/main/scala/akka/stream/alpakka/googlecloud/bigquery/model/QueryJsonProtocol.scala

```diff
@@ -103,15 +121,19 @@ object QueryRequest {
              timeout: util.Optional[Duration],
              dryRun: util.Optional[lang.Boolean],
              useLegacySql: util.Optional[lang.Boolean],
-             requestId: util.Optional[String]) =
+             requestId: util.Optional[String],
+             location: util.Optional[String],
+             maximumBytesBilled: util.OptionalLong) =
```

Comment:
You should not add new parameters here, it is not backwards compatible. 
The `withXyz` methods should be used to set them.